hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthCondition.scala

```diff
@@ -0,0 +1,153 @@
+package org.broadinstitute.monster.dap.healthcondition
+
+import enumeratum.values.{LongEnum, LongEnumEntry}
+
+sealed abstract class HealthCondition(
+  override val value: Long,
+  val label: String,
+  val conditionType: HealthConditionType,
+  val abbreviation: String,
+  val hasCg: Boolean,
+  val hasDx: Boolean,
+  val isOther: Boolean = false,
+  val cgPrefixOverride: Option[String] = None,
+  val computeGate: String => String = identity
+) extends LongEnumEntry
+
+object HealthCondition extends LongEnum[HealthCondition] {
+  override val values = findValues
+
+  import HealthConditionType.{findValues => _, _}
+
+  // scalafmt: { maxColumn = 140, newlines.topLevelStatements = [] }
```

Comment:
Wow, this is just changing the formatting for this section?... Pretty neat! I didn't know that was possible.